models/balancer/ethereum/balancer_ethereum_schema.yml

```diff
@@ -193,4 +193,37 @@ models:
       - name: vebal_balance
         description: "Amount of veBAL"
       - name: lock_time
-        description: "Amount of time the BPT was locked for at the last time the lock was updated"
\ No newline at end of file
+        description: "Amount of time the BPT was locked for at the last time the lock was updated"
+
+  - name: balancer_vebal_slopes
+    meta:
+      blockchain: ethereum
+      project: balancer
+      contibutors: markusbkoch, mendesfabio, stefenon
+    config:
+      tags: ['balancer', 'ethereum', 'vebal', 'slope', 'markusbkoch', 'mendesfabio', 'stefenon']
+    description: >
+        Slope and bias of veBAL per wallet after each balance update
+        Depends on veBAL_call_create_lock, veBAL_evt_Deposit and veBAL_evt_Withdraw
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_number
+            - wallet_address
+    columns:
+      - *block_number
+      - *block_time
+      - name: block_timestamp
+        description: "Block timestamp"
+      - name: wallet_address
+        description: "Address of the wallet holding the veBAL"
+      - name: bpt_balance
+        description: "Amount of BPT held in the veBAL lock"
+      - name: unlocked_at
+        description: ""
+      - name: slope
+        description: ""
+      - name: bias
+        description: ""
+      - name: block_date
+        description: "Block time trunc to date"
```

Comment:
Hi @Stefenon, in my last commit I try to match the columns in schema.yml to vebal_slopes.sql. If you know the detailed description of these columns, you can fill in the description field.

Reply:
@Hosuke hey I just added the descriptions you asked -- can we proceed to merge this PR?

Reply:
The PR will be merged soon (after the merge freeze).